string bt_serial
uint16 bt_position
uint16 tdc_data
uint16 tac_data
uint16 pdc_data
uint16[22] pac_data
uint16[19] electrode_data
